src/components/SingleArticle.jsx

```diff
@@ -11,13 +11,18 @@ const SingleArticle = () => {
     const [comments, setComments] = useState([]);
     const [showComments, setShowComments] = useState(true);
     const [error, setError] = useState(false);
```

Comment:
What is this error state for? Do you really need two error states for a single article page?